### PR TITLE
Add tests to verify RazorPages works with EmbeddedFileProvider

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
Realized we didn't have any coverage for this when looking at https://github.com/aspnet/Mvc/issues/6489. It works, but because how `EmbeddedFileProvider` works (or rather does not), it only supports pages with a content root at "/"

File: Mvc.sln [hunk 1]
```diff
@@ -122,6 +122,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Mvc.Pe
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Mvc.Testing", "src\Microsoft.AspNetCore.Mvc.Testing\Microsoft.AspNetCore.Mvc.Testing.csproj", "{7500B228-1769-4CFB-A571-3DFAC6678A06}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "EmbeddedPagesClassLibrary", "test\WebSites\EmbeddedPagesClassLibrary\EmbeddedPagesClassLibrary.csproj", "{3BCD5918-ABAF-46AB-BFAC-9FE097875435}"
+EndProject
```
Comment (rynowak) [inline Mvc.sln]: :(

File: test/WebSites/RazorPagesWebSite/RazorPagesWebSite.csproj [hunk 2]
```diff
@@ -7,11 +7,17 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <EmbeddedResource Include="EmbeddedPages\**" />
+  </ItemGroup>
```
Comment (rynowak) [inline test/WebSites/RazorPagesWebSite/RazorPagesWebSite.csproj]: Shouldn't these be embedded into the other library? If not then what's the point of it

Comment (pranavkm) [inline test/WebSites/RazorPagesWebSite/RazorPagesWebSite.csproj]: Oops, this needs to be removed.

**Comment by pranavkm** [comment]
🆙 📅

**Comment by pranavkm** [comment]
Decided not to take this since we wouldn't recommend EmbeddedFileProvider for Razor Pages given it's limitations.

**Pull request closed without merging**